CMakeLists.txt

```diff
@@ -115,10 +114,7 @@ if(VERBOSE)
 	message(STATUS "contrib_private_headers: ${contrib_private_headers}")
 endif()
 
-include_directories(${YAML_CPP_SOURCE_DIR}/src)
-include_directories(${YAML_CPP_SOURCE_DIR}/include)
-
-
+include_directories(include)
```

Comment:
I'm not sure this is correct; `include_directories` is usually passed an absolute path. At best, this change is superfluous.